src/node_report.h

```diff
@@ -0,0 +1,168 @@
+#ifndef SRC_NODE_REPORT_H_
+#define SRC_NODE_REPORT_H_
+
+// #include "nan.h"
+#include "v8.h"
+#include <uv.h>
+#include <node.h>
+#include <node_buffer.h>
+#include <node_object_wrap.h>
+#include <algorithm>
+#include <cstring>
+#include <climits>
+#include <cstdlib>
+#include <utility>
+#if defined(_MSC_VER)
+# pragma warning(push)
+# pragma warning(disable : 4530)
+# include <queue>
+# include <string>
+# include <vector>
+# pragma warning(pop)
+#else
+# include <queue>
+# include <string>
+# include <vector>
+#endif
+
+
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <iostream>
+#include <sstream>
+#include <iomanip>
+
+#ifdef _WIN32
+#include <time.h>
+#else
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/time.h>
+#endif
+
+namespace nodereport {
+
+using v8::Isolate;
+using v8::Local;
+using v8::Message;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::Object;
+using v8::Number;
+using v8::String;
+using v8::Value;
+using v8::StackTrace;
+using v8::StackFrame;
+using v8::MaybeLocal;
+
+// Bit-flags for node-report trigger options
+#define NR_EXCEPTION  0x01
+#define NR_FATALERROR 0x02
+#define NR_SIGNAL     0x04
+#define NR_APICALL    0x08
+
+// Maximum file and path name lengths
+#define NR_MAXNAME 64
+#define NR_MAXPATH 1024
+
+enum DumpEvent {kException, kFatalError, kSignal_JS, kSignal_UV, kJavaScript};
+
+#ifdef _WIN32
+typedef SYSTEMTIME TIME_TYPE;
+#else  // UNIX, OSX
+typedef struct tm TIME_TYPE;
+#endif
+
+// NODEREPORT_VERSION is defined in binding.gyp
+#if !defined(NODEREPORT_VERSION)
+#define NODEREPORT_VERSION "dev"
+#endif
```

Comment:
`NODEREPORT_VERSION` is unused. Please remove.

Reply:
removed